hibernate-envers/src/main/java/org/hibernate/envers/configuration/EnversSettings.java

```diff
@@ -142,4 +143,17 @@
 	 * @since 5.4.7
 	 */
 	String MODIFIED_COLUMN_NAMING_STRATEGY = "org.hibernate.envers.modified_column_naming_strategy";
+
+	/**
+	 * Globally defines whether legacy relation not-found behavior should be used or not.
+	 * Defaults to {@code true}.
+	 *
+	 * By specifying {@code true}, any {@code EntityNotFoundException} will be thrown unless the containing
+	 * class or property explicitly specifies that use case to be ignored.  Conversely, when specifying the
+	 * value {@code false}, the inverse applies an requires explicitly specifying the use case as error so
```

Comment:
seems 'the inverse applies an requires explicity specifiying ...' is confusing. 'an' -> 'and'?